lib/identifiers/isbn.rb

```diff
@@ -27,7 +27,7 @@ def self.extract_ten_digit_isbns(str)
     end
 
     def self.isbn_13_check_digit(isbn)
-      sum = isbn.each_char.zip([1, 3].cycle).reduce(0) { |sum, values| sum + (Integer(values[0]) * values[1]) }
+      sum = isbn.each_char.zip([1, 3].cycle).reduce(0) { |acc, values| acc + (Integer(values[0]) * values[1]) }
```

Comment:
#10 answers that question. I've pulled out any ISBN changes and the version bump from this PR so we can deal with those separately.